YouTubeDeepDarkMaterial.user.css

```
@@ -3144,19 +3149,35 @@
 		border-color: var(--hover-background) !important;
 	}
 	/*Contents - account*/
-	.account-page-header
+	.account-page-header,
+	ytd-section-list-renderer[page-subtype="account-settings"] a.yt-simple-endpoint.yt-formatted-string
 	{
 		color: var(--main-color) !important;
 	}
 	#account-page-header-title, h2.account-section-header
 	{
 		color: var(--main-text) !important;
 	}
+	/*title of the page*/
+	#name.ytd-page-introduction-renderer,
+	#name.ytd-channel-options-renderer,
+	ytd-settings-options-renderer .yt-formatted-string:not(a).bold
+	{
+		color: var(--main-text) !important;
+	}
 	.exp-kevlar-settings #account-page-header-subtitle, .account-info-item .account-info-label,
 	.account-section .account-email, .account-content, .account-section-subtext
 	{
 		color: var(--dimmer-text) !important;
 	}
+	/*other text*/
+	#text.ytd-page-introduction-renderer, /*email*/
+	#text.ytd-settings-options-renderer, /*descriptions, metadata, below text*/
+	#label.ytd-settings-checkbox-renderer, #label.ytd-settings-radio-option-renderer /*playback*/
```

A comma is missing here.
```suggestion
	#label.ytd-settings-checkbox-renderer, #label.ytd-settings-radio-option-renderer, /*playback*/
```